src/go/k8s/pkg/resources/statefulset_update.go

```diff
@@ -267,14 +329,25 @@ func (r *StatefulSetResource) podEviction(ctx context.Context, pod, artificialPo
 		return err
 	}
 
-	var ordinal int64
+	var ordinal int32
 	ordinal, err = utils.GetPodOrdinal(pod.Name, r.pandaCluster.Name)
 	if err != nil {
 		return fmt.Errorf("cluster %s: cannot convert pod name (%s) to ordinal: %w", r.pandaCluster.Name, pod.Name, err)
 	}
 
-	if patchResult.IsEmpty() {
-		if err = r.checkMaintenanceMode(ctx, int32(ordinal)); err != nil {
+	managedDecommission, err := r.IsManagedDecommission()
+	if err != nil {
+		log.Error(err, "not performing a managed decommission")
```

Comment:
Why error?
```suggestion
		log.Error(err, "not performing a managed decommission")
```

Reply:
Because we're not performing a managed decommission do to an error. This is important to call out in the log.

Reply:
`Info` will be call out too. I don't think this is error to not perform managed decommission. I would even say managed decommission is much more rare event than normal rolling update